packages/nimble-components/src/table/tests/table-sorting.spec.ts

```diff
@@ -733,4 +733,165 @@ describe('Table sorting', () => {
             expect(getRenderedRecordIds()).toEqual(['3', '4', '1', '2']);
         });
     });
+
+    describe('interactive sorting', () => {
+        const data: readonly SimpleTableRecord[] = [
+            {
+                id: '1',
+                stringData1: 'a1',
+                stringData2: 'b1',
+                stringData3: 'c1'
+            },
+            {
+                id: '2',
+                stringData1: 'a2',
+                stringData2: 'b2',
+                stringData3: 'c2'
+            },
+            {
+                id: '3',
+                stringData1: 'a3',
+                stringData2: 'b3',
+                stringData3: 'c3'
+            },
+            { id: '4', stringData1: 'a4', stringData2: 'b4', stringData3: 'c4' }
+        ] as const;
+
+        async function shiftClickOnElement(
```

Comment:
Could you move this helper to the page object and expose `clickOnHeader() and `shiftClickOnHeader()` from there? Ideally the page object wouldn't expose the header element on its API at all but I'm not sure how many places you'd need to clean up to make `getHeaderElement()` private.

Reply:
Updated.

I left getHeaderElement on the page object for now, though I agree it's a bit strange to be on a page object. Currently the table-sorting tests are using it to verify both public and internal sorting state on columns (`sortDirection`, `firstSortedColumn`). I guess it could be a helper method in that test file but then we'd be duplicating the CSS selectors (both in the test file and the page object).